app/controllers/logs_controller.rb

```diff
@@ -292,6 +296,7 @@ def take
         redirect_to :back
```

Comment:
I know this wasn't a part of the PR but `redirect_to :back` should probably be phased out, as not all browsers send `HTTP_REFERER`. Could rescue `ActionController::RedirectBackError` but that doesn't seem worth it. http://blog.bigbinary.com/2016/02/29/rails-5-improves-redirect_to_back-with-redirect-back.html